.eslintrc.json

```diff
@@ -0,0 +1,108 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true,
+    "mocha": true
+  },
+  "globals": {
+    "app": true,        // BitGo side-effect from testutil
+    "ethUtil": true     // BitGo side-effect from testutil
+  },
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 8
+  },
+  "rules": {
+    "indent": ["error", 2, {"SwitchCase": 1, "MemberExpression": "off"}],
+    "linebreak-style": ["error", "unix"],
+    "semi": ["error", "always"],
+    "eqeqeq": ["warn", "always"],
+    "curly": "error",
+    "no-extra-boolean-cast": "error",
+    "object-curly-spacing": ["error", "always", {"objectsInObjects": true, "arraysInObjects": true}],
+    "array-bracket-spacing": ["error", "never"],
+    "require-yield": "off",
+    "func-call-spacing": ["error", "never"],
+    "quotes": ["error", "single", { "avoidEscape": true, "allowTemplateLiterals": true }],
+    "key-spacing": ["error", { "beforeColon": false, "afterColon": true, "mode": "strict" }],
+    "quote-props": ["error", "as-needed"],
+    "no-console": "off",
+    "no-cond-assign": "error",
+    "no-empty": ["warn", { "allowEmptyCatch": false }],
```

Comment:
Do we want this to error instead? Can't think of any place where empty conditionals would be desirable

Reply:
done, I just copied this whole config from platform's